easybuild/tools/toolchain/compiler.py

```diff
@@ -88,6 +88,8 @@ class Compiler(Toolchain):
         'static': (False, "Build static library"),
         '32bit': (False, "Compile 32bit target"),  # LA, FFTW
         'openmp': (False, "Enable OpenMP"),
+        'vectorize': (None, "Enable compiler auto-vectorization, default except for noopt and lowopt"),
+        'novectorize': (None, "Disable compiler auto-vectorization, default for noopt and lowopt"), # not set, only used to map
```

Comment:
@bartoldeman Why do we need both? Can't we detect `True` or `False` (vs `None`) for `vectorize` instead?

Reply:
the problem is in the mappings in gcc.py etc. I cannot map 'vectorize' to '-ftree-vectorize' for True and to '-fno-tree-vectorize' for False, unless those mappings get redesigned of course.

Reply:
Hmm. I could use `['fno-tree-vectorize','ftree-vectorize']` with a bit of code to select first or second. It's just inconsistent with the current use. Maybe use a dict to distinguish:
`{False: 'fno-tree-vectorize', True: 'ftree-vectorize'}`